### ISSUE TITLE
zulip 1.5.1 : No module named django.utils.crypto

### ISSUE THREAD

**Issue by matclab** [galexrt/docker-zulip#76]
After configuring and doing a `docker-compose up` I get the following error:

```
zulip_1      | Generating Zulip secrets ...
zulip_1      | Traceback (most recent call last):
zulip_1      |   File "/root/zulip/scripts/setup/generate_secrets.py", line 18, in <module>
zulip_1      |     from django.utils.crypto import get_random_string
zulip_1      | ImportError: No module named django.utils.crypto
zulip_zulip_1 exited with code 1
```

**Comment by galexrt** [comment]
This is fixed in the new release `1.5.1-2`.
The image is currently built. Image build should take about 10 minutes.
Please switch to the new release tag.

**Comment by matclab** [comment]
Thanks a lot !
